keyboards/gmmk/pro/ansi/keymaps/jqs/layout.vial

```diff
@@ -0,0 +1 @@
 "KC_BSLASH", "KC_F11", "KC_ENTER", "KC_F12", "KC_F9", "KC_F10"]], [["KC_TRNS", -1, -1, "RGB_SPD", "KC_TRNS", "RGB_SPI", "KC_TRNS", "KC_TRNS"], ["RGB_HUI", "KC_TRNS", "RGB_HUD", "RGB_TOG", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["RGB_SAI", "KC_TRNS", "RGB_SAD", "KC_NO", "KC_TRNS", "KC_TRNS", "KC_MYCM", "KC_TRNS"], ["RGB_VAI", "KC_CALC", "RGB_VAD", "KC_MSEL", "KC_TRNS", "RGB_MOD", "KC_WHOM", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "RGB_RMOD", "KC_TRNS", "OSL(0)", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_NO", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "MAGIC_TOGGLE_NKRO", "KC_TRNS", -1, 23553], [-1, "KC_TRNS", "RESET", "KC_MPLY", "KC_TRNS", "KC_MSTP", "KC_MPRV", "KC_MNXT"]], [["KC_TRNS", -1, -1, "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_NO", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_NO", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", -1, "KC_TRNS"], [-1, "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"]], [["KC_NO", -1, -1, "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO"], ["KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO"], ["KC_NO", "KC_NO", "KC_NO", "KC_NO", 11050, "KC_ENTER", "KC_NO", "KC_NO"], ["KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", 10496, "KC_NO", "KC_NO"], [7724, 8244, 8737, 13348, 10022, 11813, 11574, 14391], [7719, 8223, 8737, 9251, 9765, 13107, 11830, 14391], ["LALT(KC_2)", "LSA(KC_B)", "LGUI(KC_D)", "SGUI(KC_F)", 3083, 3597, 4111, "RSFT(KC_N)"], ["RALT(KC_P)", 5653, "RGUI(KC_T)", 6681, 7195, 12061, 12337, 11555], ["LALT(KC_GRAVE)", "LSA(KC_B)", "LGUI(KC_D)", "SGUI(KC_F)", 3083, 3597, 4111, "RSFT(KC_N)"], ["RALT(KC_P)", 5653, "RGUI(KC_T)", 6681, 7195, 12061, -1, 19509], [-1, "KC_NO", "HYPR(KC_FIND)", 54272, 65535, 51199, "KC_NO", "RGUI_T(KC_NO)"]]], "encoder_layout": [[], [], [], []], "layout_options": 0, "macro": [[], [], [], [], [], [], [], [], [], [], [], [], [], [], [], []], "vial_protocol": -1, "via_protocol": 9, "tap_dance": [], "combo": [], "settings": {}}
```

Comment:
This appears to be a VIAL json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware.